releasewarrior/templates/firefox/release-rc.json.tmpl

```diff
@@ -28,30 +34,30 @@
                 "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Release-Promotion-Tasks#obtain-sign-offs-for-changes",
```

Comment:
This needs to point to https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto-rc.md#obtain-sign-offs-for-changes instead